### ISSUE TITLE
Meeting Agenda 09/05/25

### ISSUE THREAD

**Issue by jerbeario** [jerbeario/MasterThesis#7]
Hi @maartenmarx 

# Updates
## Binary Classification
ive chosen to go for a binary classification approach where the output of the model before thresholding is used as the susceptibility score. This is what has been done in the literature

## Spatial Partitioning
I created a [map to partition the data](
![](https://github.com/jerbeario/MasterThesis/blob/main/Input/Europe/partition_map/balanced_partition_map.png)
) into training testing and validation. Ive done it in such a way that each country is approximately split into 70-15-15 train, test, valid. This map can also be used to only train on countries where we have labelled data for. **please motivate the chosen stratification in your sampling method**

# Agenda

## Class Imbalance 
Currently ensuring class balance in the train set by under-sampling the majority class. Im wondering what to do for my validation and testing sets. Do i also balance the classes in them? My gut tells me no because we want to test the model on the real distribution. But then the test and valid sets are huge compared to the training set because 80% of training set was thrown away. See the [full balanced](
![](https://github.com/jerbeario/MasterThesis/blob/main/Input/Europe/partition_map/full_balanced_partition_map_sub_countries.png)
) and only [trained balanced map](
![](https://github.com/jerbeario/MasterThesis/blob/main/Input/Europe/partition_map/balanced_partition_map.png)
)
**good point, please follow the literature on class balance sampling. The priors will be very different of course. You may also approach this experimentally, and step by step shrink the test set to the same ratio as you have in train.**

## Reduced Scope
Given the time thats left, it might be ambitious to do the multi-hazard modelling. Once I finish the single hazard maps i might have the time to put together a quick model but i might not have time to to SHAP and hyper parameter optmization. **Please keep on doing this. It is a master thesis.**

## Results
Historically, Ive always struggled writting the results section. I obviously want to include a PR curve for each model (RF, MLP, CNN) and probably also compare F1 scores. Do you have any tips/advice for structuring the results ? **Oh yes, see my "how to work with marx, and also the thesis template you use. The most imporatnt thing is that the research questions are super good. Best is a hierarchical setup. Then you structure the thesis according to the hierarchy of the RQs. For each sentence in the thesi it has to be clear *what is is for*, ie for which RQ. And vioce versa, you must of course pay attention (and answer) each RQ. This really helps, it helps you as an author, and helps the reader and most importantly your examiner who wants to quickly get a good view of the research that you did.**
